exercises/custom-converter/practice/src/activities.ts

```diff
@@ -0,0 +1,10 @@
+import * as activity from '@temporalio/activity';
+import { ApplicationFailure } from '@temporalio/activity';
+
+export async function converterActivity(input: string): Promise<string> {
+  const context = activity.Context.current();
+  context.log.info('User is successfully logged', { input });
```

Comment:
```suggestion
  activity.log.info('User is successfully logged', { input });
```

and remove the Context.current() on the previous line.